npm/vue/src/index.ts

```diff
@@ -323,7 +323,21 @@ export function mount<
   >
 >
 
-// implementation
+/**
+ * Mounts a component and returns an object containing the component and VueWrapper
+ * @param componentOptions
+ * @param options
+ * @returns {{vueWrapper: VueWrapper, component: ComponentPublicInstance}}
```

Comment:
Could we wrap this return as a type named MountResponse?

Reply:
I'm going to revert this to the type it is actually returning currently `Cypress.Chainable<VueWrapper>`. There is a card to change this, but I shouldn't have added that here 😬